playbooks/agent_driver_deploy.yaml

```diff
@@ -0,0 +1,44 @@
+---
+# version: 2016-12-29
+- hosts: all
+  vars_files:
+    - vars/agent_driver_deploy_vars.yaml
+  become_user: root
+  become_method: sudo
+  become: true
+  remote_user: '{{ remote_user }}'
+  roles:
```

Comment:
We need to be able to locate the agent on a different host than the driver (neutron server).  Can we either split driver and agent apart into separate files, or somehow provide the 'roles' on a per host basis?